server/test/unit/buttonAlertsTest/handleValidRequestTest.js

```diff
@@ -627,6 +627,9 @@ describe('buttonAlerts.js unit tests: handleValidRequest', () => {
       it('should not send an alert session update', async () => {
         expect(this.sendAlertSessionUpdateStub).not.to.be.called
       })
+
+     
+
```

Comment:
unnecessary whitespace?

Reply:
Fixed this, I'm not sure why that happened.